lighthouse-core/report/scripts/lighthouse-report.js

```diff
@@ -196,26 +197,31 @@ class LighthouseReport {
   }
 
   /**
-   * Sets up listeners to expand audit `<details>` when the user prints the page.
+   * Expands audit `<details>` when the user prints the page.
    * Ideally, a print stylesheet could take care of this, but CSS has no way to
-   * open a `<details>` element. When the user closes the print dialog, all
-   * `<details>` are collapsed.
+   * open a `<details>` element.
    */
-  _setupExpandDetailsWhenPrinting() {
+  expandDetailsWhenPrinting() {
+    const details = Array.from(document.querySelectorAll('details'));
+    details.map(detail => detail.open = true);
+  }
+
+  /**
+   * Sets up listeners to collapse audit `<details>` when the user closes the
+   * print dialog, all `<details>` are collapsed.
+   */
+  _setUpCollaspeDetailsAfterPrinting() {
     const details = Array.from(document.querySelectorAll('details'));
 
     // FF and IE implement these old events.
     if ('onbeforeprint' in window) {
-      window.addEventListener('beforeprint', _ => {
```

Comment:
one thing we loose out on with this is `ctrl+p` (or printing from the browser chrome) no longer expands the dropdowns. Meaning users have to go through the "Export" dropdown for printing. That might be unavoidable though b/c of the bug this PR is trying to address.